### PR TITLE
grade app ui changes

### PR DESCRIPTION
**Pull request by janno42**
this changes the following:
- show in form title whether uploading final or midterm grades
- hide last edited fields for initial upload
- add missing translations
- remove form field description
- show on staff page how many grade documents have been uploaded

please post other requests here.

### CODE REVIEW

File: evap/grades/forms.py [hunk 1]
```diff
@@ -35,18 +35,20 @@ def __init__(self, *args, **kwargs):
...
+        self.fields['last_modified_user_2'].widget.attrs['readonly'] = "True"
+        self.fields['last_modified_time_2'].initial = self.instance.last_modified_time
         if self.instance.last_modified_user:
             self.fields['last_modified_user_2'].initial = self.instance.last_modified_user.full_name
-        self.fields['last_modified_user_2'].widget.attrs['readonly'] = "True"
-
+        else:
+            self.fields['last_modified_time_2'].widget = forms.HiddenInput()
+            self.fields['last_modified_user_2'].widget = forms.HiddenInput()
+
```
Comment (karyon) [inline evap/grades/forms.py]: how about moving the stuff above the if into the if? something like
```
if ...:
    set initial values
    set readonly to True
else:
    make them hidden

File: evap/grades/templates/grades_semester_view.html [hunk 2]
```diff
@@ -48,8 +48,8 @@ <h3 class="panel-title">{% trans "Courses" %}</h3>
                                     <div class="btn-group">
                                         <button type="button" class="btn btn-sm btn-default dropdown-toggle" data-toggle="dropdown" aria-expanded="false" {{ disable_if_archived }}>{% trans "Upload grades" %} <span class="caret"></span></button>
                                         <ul class="dropdown-menu" role="menu">
-                                            <li><a href="{% url "grades:upload_grades" semester.id course.id %}">Midterm grades</a></li>
-                                            <li><a href="{% url "grades:upload_grades" semester.id course.id %}?final=true">Final grades</a></li>
+                                            <li><a href="{% url "grades:upload_grades" semester.id course.id %}">{% trans "Midterm grades" %}</a></li>
+                                            <li><a href="{% url "grades:upload_grades" semester.id course.id %}?final=true">{% trans "Final grades" %}</a></li>
```
Comment (karyon) [inline evap/grades/templates/grades_semester_view.html]: i absolutely did not check for missing i18n during review, sorry for that. i'll try to remember next time

File: evap/staff/templates/staff_semester_view_course.html [hunk 3]
```diff
@@ -60,11 +60,16 @@
...
                     <span class="glyphicon glyphicon-comment"></span>
                     <span>{{ course.num_textanswers }}</span>
-                </p>
+                </span>
             {% endif %}
+            <br>
+            <span data-toggle="tooltip" data-placement="left" title="{% trans "Grade documents (Midterm, Final)" %}">
+                <span class="glyphicon glyphicon-file"></span>
+                <span>{% blocktrans with midterm=course.midterm_grade_documents.count final=course.final_grade_documents.count %}M: {{ midterm }}, F: {{ final }}{% endblocktrans %}</span>
+            </span>
```
Comment (karyon) [inline evap/staff/templates/staff_semester_view_course.html]: could you move these changes into a separate commit or at least mention it in the commit message?

**Comment by janno42** [comment]
updated

**Comment by karyon** [comment]
:100:
